### DISCUSSION TITLE
Changing boundary condition

### DISCUSSION THREAD

**Discussion by deepakbiitk** [idaholab/moose#24715] [Q&A General] [answered]
Hello MOOSE users,

I am solving heat equation in a rectangular domain, In my left, bottom and top boundary I have insulated bc and in the right boundary I have convection bc for few time steps after which I would like to change it to DirichletBC for the rest of my simulation. Is it possible to achieve in a single input file or do I need to use the restart feature? 

Thanks,
Deepak

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please see this thread
https://github.com/idaholab/moose/discussions/24596
You ll want to use the Control system to try to do this.
If it doesnt work, then restart will be the way

Guillaume

**Reply by deepakbiitk** [reply]
Hi Guillaume,

Use of control system worked  in this case. I appreciate your suggestion.

Deepak
